recipes/glsl-optimizer/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GLSLOptimizerConan(ConanFile):
+    name = "glsl-optimizer"
+    description = "GLSL optimizer based on Mesa's GLSL compiler. Used in Unity for mobile shader optimization."
+    homepage = "https://github.com/aras-p/glsl-optimizer"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "glsl", "opengl")
+    license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("opengl/system")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob(self.name + "-*/")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        cmake = CMake(self)
```

Comment:
It looks like the build is failing using msvc because of a missing getopt.
cci has a `getopt-for-visual-studio` recipe for that.